docker-compose.yml

```diff
@@ -21,6 +20,7 @@ services:
       - database:database
     ports:
       - 8000:8000
+      - 4444:4444
```

Comment:
Why we are exposing this?

Reply:
It's for remote_pdb, installed here https://github.com/CodeForPoznan/pah-fm/blob/9ac2f80103dbc6fe4ac6643f12495ddc34680ed6/backend/Pipfile#L37
`import remote_pdb; remote_pdb.set_trace(host='0.0.0.0', port=4444)`

Reply:
To prevent from exposing some forgotten remote pdbs (it's hardly possible) I've introduced targets in backend image: backend-runtime for production and backend-dev for development. See details 403d4b9106a9aa1ee2648bfd56c31542d99c2435

Reply:
That change would need to be reflected in build steps as well. AFAIR after merging this one this will be exposed on prod.

Have you checked `debug-backend` command from Makefile? I just haven't used `remote_pdb`, but I've seen that Artur has added it some time ago.

Reply:
I've introduced production docker compose where we use backend-runtime stage of backend image. It prevents from exposing 4444.The image is built by the new `make build-backend-runtime` target. See more details in my last commit. `debug-backend` is something different than `remote_pdb` @w1stler